.circleci/config.yml

```diff
@@ -0,0 +1,62 @@
+version: 2.1
+
+# Define a job to be invoked later in a workflow.
+# See: https://circleci.com/docs/2.0/configuration-reference/#jobs
+jobs:
+  test:
+    docker:
+      - image: cimg/clojure:1.10
```

Comment:
Why not `cimg/clojure:1.10.3-openjdk-11.0` to match a more specific Clojure version and target JDK ?

That would then match the exact versions used in the Docker image specified in the `deploy` section.

Reply:
Thanks for spotting will look at this...